zest/releaser/pypi.py

```diff
@@ -407,6 +407,31 @@ def version_levels(self):
             return default
         return result
 
+    def tag_format(self):
+        """Return the string formating the tag name in the release.
+
+        Override the default ``%(version)`` in setup.cfg using
+        a ``tag-format`` option::
+
+            [zest.releaser]
+            tag-format = v%(version)s
+
+        Returns default of ``%(version)s`` when nothing has been configured.
+        """
+        default = '%(version)s'
+        if self.config is None:
+            return default
+        try:
+            result = self.config.get(
+                'zest.releaser', 'tag-format')
+            try:
+                '%(version)s'%({'version':''})
+            except KeyError:
+                raise ValueError('version needs to be part of the string formater use default')
```

Comment:
Note: check the rest of the code on how we print something before a "sys.exit", I don't remember if we use logging or `print()` right now :-)